dbt/include/fabric/macros/materializations/models/incremental/incremental.sql

```diff
@@ -3,21 +3,15 @@
 
   {%- set full_refresh_mode = (should_full_refresh()) -%}
   {% set target_relation = this.incorporate(type='table') %}
-  {%- set relations_list = fabric__get_relation_without_caching(target_relation) -%}
+  {%- set relation = adapter.get_relation(database=this.database, schema=this.schema, identifier=this.identifier) -%}
```

Comment:
Exactly the same, just more concise:
```suggestion
{%- set relation = load_cached_relation(this) -%}
```